_articles/custom-mouse-buttons.md

```diff
@@ -1,20 +1,51 @@
 ---
 layout: article
-title: Setting Custom Mouse Actions
+title: Customizing Your Mouse
 description: >
-   How to set custom mouse actions with xbindkeys
+   How to customize the buttons on your mouse
 keywords:
   - Support
   - Pop
   - Ubuntu
   - xbindkeys
+  - Solaar
+  - Piper
 image: http://support.system76.com/images/system76.png
 hidden: false
 section: software-applications
 
 ---
 
-### Software
+### Graphical Software
+
+#### Piper
+
+![Piper Buttons Window](/images/custom-mouse-buttons/Piper-2.png) 
+
+<u>Piper</u> is a frontend to the <u>libratbag</u> backend for setting custom actions to mouse buttons like the Logitech MX Master 3 mouse that we sell and this section will be using that mouse as an example for using <u>Piper</u>. You can see the source code of <u>Piper</u> [here](https://github.com/libratbag/piper).
```

Comment:
like the Logitech MX Master 3 mouse that we sell

Change to "like the Logitech MX Master 3 (sold by System76). In this section we will be...

Reply:
This is still a run-on sentence.